From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Tue, 15 Nov 2022 21:48:17 +0100
Subject: docs: Use Sphinx RTD theme instead of Pylons one

The Pylons Sphiny theme isn't packaged yet, lets use the classical RTD
theme then.

Forwarded: Not-Needed
---
 docs/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 9e5e04f..c8cc651 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -17,7 +17,7 @@ try:
     import importlib.metadata as importlib_metadata
 except ImportError:
     import importlib_metadata
-import pylons_sphinx_themes
+import sphinx_rtd_theme
 
 # If your extensions are in another directory, add it here.
 # sys.path.append('some/directory')
@@ -87,8 +87,8 @@ pygments_style = 'sphinx'
 # Options for HTML output
 # -----------------------
 
-html_theme = 'pylons'
-html_theme_path = pylons_sphinx_themes.get_html_themes_path()
+html_theme = "sphinx_rtd_theme"
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 html_theme_options = dict(
     canonical_url='https://docs.pylonsproject.org/projects/pastedeploy/en/latest/'
 )
